libcontainer/init_linux.go

```diff
@@ -140,6 +142,30 @@ func finalizeNamespace(config *initConfig) error {
 	return nil
 }
 
+// lateCgroups applies any cgroup limits which must be done as late as possible.
+// Specifically, this refers to pids.max which runs into issues when Go likes to
+// start spawning new OS threads seemingly at random.
+func lateCgroups(config *initConfig) error {
+	if path, ok := config.CgroupPaths["pids"]; ok {
+		// XXX: We have to bootstrap this here because the manager hasn't joined
```

Comment:
It actually looks like the `MkdirAll` isn't necessary? It seems to work without it, but it didn't work when I added it. I must've rebased or something because the manager does `Apply` when it should.